tests/scenarios/traffic_split_with_apex_service_test.go

```diff
@@ -129,9 +135,9 @@ var _ = Describe(``+
 
 				expectedWeightedCluster := &xds_route.WeightedCluster{
 					Clusters: []*xds_route.WeightedCluster_ClusterWeight{
-						weightedCluster("bookstore-v2", 10),
+						weightedCluster("bookstore-v2", 100),
```

Comment:
Curious to know why we changed these weights

Reply:
This is because in routes v2 we build the outbound policies with default weights : https://github.com/openservicemesh/osm/blob/8a19278719861d9adf13c1a1b82418592c4c040a/pkg/catalog/routes.go#L695

The weights will only get updated for outbound virtual hosts that is referenced in a traffic split, otherwise it always remains 100. This is different from routes v1 and hence the change in weights